packages/vscode-extension/src/handlers.ts

```diff
@@ -1410,6 +1404,33 @@ export async function showLocalDebugMessage() {
   });
 }
 
+export async function getLocalDebugMessageTemplate(isWindows: boolean): Promise<string> {
```

Comment:
Could these 2 functions be moved to another file like `utils/*.ts`? I'm trying to keep the `handlers.ts` small.